src/webapi/document.rs

```diff
@@ -67,4 +69,54 @@ impl Document {
             ).into_reference_unchecked()
         }
     }
+
+    /// Returns the <body> or <frameset> node of the current document, or null if no such element exists.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/body)
+    // https://html.spec.whatwg.org/#the-document-object:dom-document-body
+    pub fn body( &self ) -> Option< HtmlElement > {
+        unsafe {
+            js!(
+                return @{self}.body;
+            ).into_reference_unchecked()
+        }
+    }
+
+    /// Returns the <head> element of the current document. If there are more than one <head>
+    /// elements, the first one is returned.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/head)
+    // https://html.spec.whatwg.org/#the-document-object:dom-document-head
+    pub fn head( &self ) -> Option< HtmlElement > {
+        unsafe {
+            js!(
+                return @{self}.head;
+            ).into_reference_unchecked()
+        }
+    }
+
+    /// Gets the title of the document.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/title)
+    // https://html.spec.whatwg.org/#the-document-object:document.title
+    pub fn title( &self ) -> String {
+        unsafe {
+            js!(
+                return @{self}.title;
+            ).try_into().unwrap()
+        }
+    }
+
+    /// Sets the title of the document.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/title)
+    // https://html.spec.whatwg.org/#the-document-object:document.title
+    pub fn set_title( &self, title: &str ) -> String {
```

Comment:
Ah, one last thing - we don't generally return the value itself from the setters (usually there's no point), so please get rid of the `-> String` return value.

Reply:
ah yea, I had wondered about that... I don't really see it as making a huge difference and it could be useful in some cases? I'll remove it, but I am curious what the reasoning for this is?

Reply:
> I'll remove it, but I am curious what the reasoning for this is?

The reasoning is very simple - there isn't much point in returning the *exactly* the same thing you're passing into the function. (:

```
foo.set_title( "bar" ); // <-- you already know you're passing "bar", so what's the point in returning it?
```

Besides it's always going to be slightly faster to have a function without a return value. (As then we can skip the serialization of the return value.)

Reply:
> The reasoning is very simple - there isn't much point in returning the exactly the same thing you're passing into the function. (: